docs/tasks.md

```diff
@@ -0,0 +1,23 @@
+# Bolt Tasks
+
+This repo includes bolt tasks.  The available tasks are enumerated here.
+
```

Comment:
Add generic comment about the repo needing to be the CWD for the below example commands to work

Reply:
I have added the comment in the latest update.